src/electron/main/main.ts

```diff
@@ -41,6 +40,14 @@ const createWindow = () => {
         mainWindow.show();
         enableDevMode(mainWindow);
     });
+
+    electronAutoUpdateCheck
+        .check()
+        .then(() => {
+            console.log('checked for updates');
+            setupRecurringUpdateCheck();
+        })
+        .catch(console.log);
```

Comment:
should this use `console.error`?

Reply:
Thats how it was for the `loadUrl` catch, so following same patterns!